modules/ROOT/pages/to-find-info.adoc

```diff
@@ -154,19 +224,20 @@ the reason is provided.
 |Optimized API |optimized-api |sales:/connect |Yes
 |===
 
-== Search by Tags and Categories in Exchange Cloud
 
-To search by tags and categories in Exchange Cloud, use the drop-down menus for tags and categories on the search page.
+== Search by Types, Tags, Categories, and Custom Fields in the Original Exchange Search Experience
+
+In the original Exchange search experience, you can enter search text to search for assets, click *All Types* to set a filter for specific asset types, or combine search text with the filters listed in the type menu.
```

Comment:
Would it be just as effective to use something like this?
```suggestion
In the original Exchange search experience, you can either search the names of assets or create filters to search for specific asset types, or you can combine search text with existing filters listed in the type menu.
```

Reply:
This would not be accurate. In the original search, search text may include parameters such as tags that are not asset names. Clicking *All Types* is the step to open the menu and select an existing filter but does not create a filter.